posthog/hogql_queries/web_analytics/web_analytics_query_runner.py

```diff
@@ -91,17 +95,29 @@ def session_having(self, include_previous_period: Optional[bool] = None):
         )
 
     def events_where(self):
-        properties = [
-            parse_expr(
-                "events.timestamp >= {date_from}",
-                placeholders={"date_from": self.query_date_range.date_from_as_hogql()},
-            )
-        ] + self.query.properties
+        properties = (
+            [
+                parse_expr(
+                    "events.timestamp >= {date_from}",
+                    placeholders={"date_from": self.query_date_range.date_from_as_hogql()},
+                )
+            ]
+            + self.query.properties
+            + self._test_account_filters
+        )
+
         return property_to_expr(
             properties,
             self.team,
         )
 
+    @cached_property
+    def _test_account_filters(self):
+        if isinstance(self.team.test_account_filters, list) and len(self.team.test_account_filters) > 0:
+            return self.team.test_account_filters
+        else:
+            return []
```

Comment:
I'm semi-surprised if this works considering I had to add the same functionality as:
```python
        # Filter Test Accounts
        if (
            self.query.filterTestAccounts
            and isinstance(self.team.test_account_filters, list)
            and len(self.team.test_account_filters) > 0
        ):
            for property in self.team.test_account_filters:
                filters.append(property_to_expr(property, self.team))
```

Reply:
I based this on that code :D but it should be possible to add two python arrays together if that's what you mean? Or what was it that didn't work.

Reply:
Looks like you're not doing `property_to_expr` from what I could see (unless im being blind?)

Reply:
Ah right, this happens where it's called in events_where or sessions_where